docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,175 @@
+----------------------------------------
+
+**Table of Contents**
+
+- [About Kubernetes on DCOS](#about-kubernetes-on-dcos)
+- [Prerequisites](#prerequisites)
+- [Install Kubernetes](#install-kubernetes)
+- [Operations](#operations)
+- [Uninstall Kubernetes](#uninstall-kubernetes)
+
+
+## About Kubernetes on DCOS
+
+The [Datacenter Operating System (DCOS)](https://docs.mesosphere.com/) is a collection of tools for managing distributed systems deployed on a cluster of machines (virtual or bare metal).
+
+Since Kubernetes can be configured to schedule through Mesos, the cluster kernel at the heart of DCOS, it's a natural fit to deploy Kubernetes on DCOS.
+
+The recommended way to install Kubernetes on DCOS is to use the latest pre-packaged releases in the [Mesosphere Multiverse](https://github.com/mesosphere/multiverse) DCOS package repository. This package contains a [Marathon](https://mesosphere.github.io/marathon/) deployment manifest that specifies how to run the Kubernetes core components in containers deployed and monitored by Marathon, the DCOS init system.
+
+If you want to build your own release, the package source can be found in the [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos) github repository.
+
+This tutorial will walk you through setting up Kubernetes on a DCOS cluster and starting your first pod with an nginx webserver.
+
+**NOTE:** There are [known issues with the current implementation](../../contrib/mesos/docs/issues.md).
+If you have problems completing the steps below, please [file an issue against the kubernetes-mesos project](https://github.com/mesosphere/kubernetes-mesos/issues).
+
+Further information about Kubernetes on Mesos is available in the [contrib directory](../../contrib/mesos/README.md).
+
+
+## Prerequisites
+
+- A running [DCOS cluster](https://mesosphere.com/product/)
+  - [DCOS Community Edition](https://docs.mesosphere.com/install/) is currently available on [AWS](https://mesosphere.com/amazon/), [GCE](https://mesosphere.com/google/) (early access) & [Azure](https://mesosphere.com/azure/) (coming soon).
+  - [DCOS Enterprise Edition](https://mesosphere.com/product/) can be deployed on virtual or bare metal machines. Contact sales@mesosphere.com for more info and to set up an engagement.
+- [DCOS CLI](https://docs.mesosphere.com/install/cli/) installed locally
+
+
+## Install Kubernetes
+
+1. Configure and validate the Mesosphere Multiverse as a package source repository
+
+  ```
+  $ dcos config prepend package.sources https://github.com/mesosphere/multiverse/archive/version-1.x.zip
+  $ dcos package update --validate
+  ```
+
+2. Install Etcd
+
+  By default, the Kubernetes DCOS package comes with its own single-node Etcd, but in order to survive Kubernetes component process and/or container loss, an HA Etcd cluster is required.
+  Since [Etcd](https://github.com/mesosphere/etcd-mesos) can be installed on and managed by DCOS, it is the logical choice.
```

Comment:
Suggestion: "The Kubernetes DCOS package comes with its own single-node Etcd by default. However, an HA Etcd cluster is required for your cluster (<--?) to survive the loss of a Kubernetes component or container."

Annoying question: Does Kubernetes work at all with just the default Etcd? As written, this sounds like a workaround. 

In the last sentence -- what would you be choosing among?